ScatterPeas/scatterpeas/__init__.py

```diff
@@ -0,0 +1,47 @@
+from pyramid.config import Configurator
+from sqlalchemy import engine_from_config
+from pyramid.authentication import AuthTktAuthenticationPolicy
+from pyramid.authorization import ACLAuthorizationPolicy
+import os
+from models import ReminderFactory, groupfinder, RootFactory, UserFactory, AliasFactory
+
+from .models import (
+    DBSession,
+    Base,
+    )
+
+
+def main(global_config, **settings):
+    """ This function returns a Pyramid WSGI application.
+    """
+
+    engine = engine_from_config(settings, 'sqlalchemy.')
+    DBSession.configure(bind=engine)
+    Base.metadata.bind = engine
+    auth_secret = os.environ.get('JOURNAL_AUTH_SECRET', 'itsaseekrit')
+    config = Configurator(
+        settings=settings,
+        authentication_policy=AuthTktAuthenticationPolicy(
+            secret=auth_secret,
+            hashalg='sha512',
+        ),
+        authorization_policy=ACLAuthorizationPolicy(),
+        root_factory=RootFactory
+    )
+    config.add_static_view('static', 'static')
+    config.add_route('home', '/')
+    config.add_route('login', '/login')
+    config.add_route('logout', '/logout')
+    config.add_route('list', '/list')
+    config.add_route('detail_reminder', '/detailreminder/{id}', factory=ReminderFactory, traverse='/{id}')
+    config.add_route('create_reminder', '/createreminder')
+    config.add_route('edit_reminder', '/editreminder/{id}', factory=ReminderFactory, traverse='/{id}')
```

Comment:
Thinking in a 'resource oriented' way, these URLs could be cleaned up quite a bit:
```
/reminders/  # a list of reminders for the current user?
/reminders/create  # make a new reminder
/reminders/{id}  # details of the given reminder
/reminders/{id}/edit  # edit the given reminder
```

This could be applied to url clusters below for users and aliases too.